### ISSUE TITLE
[Bug]: "Error:" "Update is older than current version Updates complete."

### ISSUE THREAD

**Issue by Francewhoa** [FreeTubeApp/FreeTube#8592]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.
- [x] I have read and agree to follow the [rules](https://docs.freetubeapp.io/community/rules/).

### Describe the bug

1. Using [Flathub](https://flathub.org/en/apps/io.freetubeapp.FreeTube), install FreeTube version before 0.23.13 Beta
1. Using Terminal/Konsole execute this command `Flatpak update`
1. Update failed with this error message: 

   > Error: While pulling app/io.freetubeapp.FreeTube/x86_64/stable from remote flathub: Update is older than current version
   > Updates complete.
1. I noticed this last week of January 18th, 2026. It seems that, since then, end-users are blocked with 0.23.13 Beta and can not install future versions of FreeTube. 

### Expected Behavior

Updating without any error message

### Issue Labels

accessibility issue

### FreeTube Version

Version released after 0.23.13 Beta

### Operating System Version

Debian 12 Bookworm

### Installation Method

Flathub

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.23.13 Beta

### Additional Information

The new version of FreeTube that cause the issue was built 10 days ago. So around January 18th, 2026. Source is the "Release notes" group at https://flathub.org/en/apps/io.freetubeapp.FreeTube or https://archive.ph/EPwUD#selection-313.0-323.19

This is a recurring challenge per https://github.com/FreeTubeApp/FreeTube/issues/7263

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by Francewhoa** [comment]
For those facing this challenge, a temporary workaround to get both the present and future updates of FreeTube is to remove the OLD FreeTube 0.23.13 Beta. Then reinstall a NEW FreeTube 0.23.13 Beta.

Steps: 

1. Using Konsole/Terminal execute those three commands:
   1. To remove OLD FreeTube 0.23.13 Beta. But keep its present configurations: 
         `flatpak remove io.freetubeapp.FreeTube`
   1. To install NEW FreeTube 0.23.13 Beta: 
         `flatpak install flathub io.freetubeapp.FreeTube`
   1. Test the above by updating the NEW FreeTube
         `flatpak update` 
   1. Error is gone.
